source/_components/sensor.llap.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: page
+title: "LLAP Sensor"
+description: "Instructions how to integrate Ciseco / Wireless Things sensors into Home Assistant."
+date: 2017-02-15 19:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Sensor
+ha_release: 0.39
+ha_iot_class: "Local Push"
+logo: wirelessthings.png
+---
+
+The `llap` platform supports wireless sensors which were produced by now defunct Ciseco / Wireless
+Things company.
+
+To communicate with the sensors you need a hardware transceiver module, like [SRF Stick][] or
+[Slice of Radio][]. This module provides a serial interface through which you can communicate with
+the sensors. There are two ways of communication: by using a simple [LLAP protocol][LLAP] directly
+through the serial interface, or by installing a [Wireless Thing Message Bridge][Message Bridge]
+on the device with hardware transciever and listening for UDP datagrams from it.
+
+The `llap` platform supports communication only through the Message Bridge. If you configured your
+Message Bridge to use a different port from the default 50140, you need to specify this port in the
+configuration. Same goes for the network name (default is 'Serial').
+
+To use LLAP sensors in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: llap
+    sensors:
+      - name: "Room Temperature"
+        id: RT
+        prefix: "TEMP"
+        unit_of_measurement: "°C"
```

Comment:
I can remove `unit_of_measurement`, but I wouldn't remove `prefix`. The reason is that in 99% of cases users will put some prefix, because except for the actual value the sensor also sends status messages (e.g., `AWAKE`, `BATTLOW`, etc.). It's optional only because someone might want to capture all messages from the sensor and process them by themselves (e.g., in template).